modules/postgresql/outputs.tf

```diff
@@ -152,3 +152,12 @@ output "env_vars" {
     "CLOUD_SQL_DATABASE_NAME" : local.database_name
   }
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//cloudsql.googleapis.com/projects${element(split("/projects", google_sql_database_instance.default.self_link), 1)}"
+    service_id  = substr(format("%s-%s", var.name, md5(var.project_id)), 0, 63)
+    location    = var.location
```

Comment:
I don't think var.location exist in this module. Use var.region for location here. Similar for mysql module. Please validate!